### ISSUE TITLE
Error unknown option "-drawcleanupcrashcmd" on simple example

### ISSUE THREAD

**Issue by ShuffleWire** [Andereoo/TkinterWeb#38]
Hello,

**The issue**
When running a minimal example of tkinterweb, I run into an issue during loading

**To reproduce**
Run `python3 -m tkinterweb.__init__`
The console says :

```
Starting TkinterWeb for 64-bit Linux with Python 3.9.2.

Traceback (most recent call last):
  File "/home/yann/.local/lib/python3.9/site-packages/tkinterweb/bindings.py", line 85, in __init__
    tk.Widget.__init__(self, master, "html", cfg, kwargs)
  File "/usr/lib/python3.9/tkinter/__init__.py", line 2572, in __init__
    self.tk.call(
_tkinter.TclError: unknown option "-drawcleanupcrashcmd"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/yann/.local/lib/python3.9/site-packages/tkinterweb/__init__.py", line 86, in <module>
    Demo()
  File "/home/yann/.local/lib/python3.9/site-packages/tkinterweb/__init__.py", line 73, in __init__
    frame = HtmlFrame(root)
  File "/home/yann/.local/lib/python3.9/site-packages/tkinterweb/htmlwidgets.py", line 30, in __init__
    self.html = html = TkinterWeb(self, message_func, HtmlFrame)
  File "/home/yann/.local/lib/python3.9/site-packages/tkinterweb/bindings.py", line 88, in __init__
    tk.Widget.__init__(self, master, "html", cfg, kwargs)
  File "/usr/lib/python3.9/tkinter/__init__.py", line 2572, in __init__
    self.tk.call(
_tkinter.TclError: unknown option "-drawcleanupcrashcmd"
```

**Current solution**
I'm manually removing the "drawcleanupcrashcmd" from kwarg, by simply removing lines 77 to 80 of the file binding.py
```
        # catch htmldrawcleanup crashes on supported platforms
        if (platform.system() == "Windows") or (platform.system() == "Linux" and sys.maxsize > 2**32):
            if "drawcleanupcrashcmd" not in kwargs:
                kwargs["drawcleanupcrashcmd"] = master.register(self.on_drawcleanupcrash)
```

**System**
Debian 11 64 bits, up to date.

Thanks for any tips,
And I can't find any references online about `drawcleanupcrashcmd` or `on_drawcleanupcrash` :/ Where did you discovered it ?

PS : good job, btw, I'm building a simple GUI and it's very usable :+1:

**Comment by Andereoo** [comment]
Hi! 

Thanks for letting me know about this.

I can't seem to reproduce this issue on my Ubuntu 20.04 64-bit system. Can you please try completely removing TkinterWeb and re-installing it? 

`drawcleanupcrashcmd` is an additional argument that is part of a couple of modifications I made to the underlying Tkhtml rendering engine that helps to minimize crashes. So far, only 64-bit Windows and Linux support it. Since `drawcleanupcrashcmd` is part of my own modifications, you wouldn't find it anywhere else on the internet.

My best guess for what is happening here is a mismatch between Tkhtml binaries, but that is only a wild guess. Let me know what happens after completely reinstalling, and we'll take it from there!

**Comment by ShuffleWire** [comment]
Ok, I understand better !

It's been actually a fresh install on a newly Debian install, made few days ago, and I got the exact same issue on both my second PC and my work PC, which are also on Debian.

Maybe some dependadancy that make your package use the "regular" tkhtml code, instad of the modifed one ?

**Comment by Andereoo** [comment]
Here's a (very) heavily simplified version of TkinterWeb: [tkinterweb.zip](https://github.com/Andereoo/TkinterWeb/files/9305330/tkinterweb.zip). Try downloading it and extracting it. Then let me know what happens!

**Comment by ShuffleWire** [comment]
Got it, extracted it, cd into tkinterweb2, then ran `python3 __init__.py`
got the same error : 
```
Traceback (most recent call last):
  File "/home/yann/Downloads/tkinterweb2/__init__.py", line 22, in <module>
    html = TkinterWeb(root)
  File "/home/yann/Downloads/tkinterweb2/__init__.py", line 13, in __init__
    tk.Widget.__init__(self, master, "html", cfg, kwargs)
  File "/usr/lib/python3.9/tkinter/__init__.py", line 2572, in __init__
    self.tk.call(
_tkinter.TclError: unknown option "-drawcleanupcrashcmd"
```

**Comment by Andereoo** [comment]
Apologies for the delay.

I installed Debian and still initially failed to reproduce this issue. I then found that if I installed the `tk-html3` package, the error `unknown option "-drawcleanupcrashcmd"` would be thrown. It seems that you already have some version of TkHtml installed on your system. When TkinterWeb tries to load TkHtml, it finds the version that is already on your system and not the modified version that is provided by TkinterWeb.

I modified the code that loads TkHtml to prioritize the files provided by TkinterWeb over those already in the system.

Try reinstalling TkinterWeb and let me know if this fixes your issue!
